.ci/jobs/apm-server-check-changelogs-mbp.yml

```diff
@@ -36,8 +33,8 @@
           recursive: true
           parent-credentials: true
           timeout: 100
+          reference-repo: /var/lib/jenkins/.git-references/apm-server.git
```

Comment:
The job in charge to refresh the cached repos is:
- [jenkins-git-fetch-reference](https://apm-ci.elastic.co/job/jenkins-git-fetch-reference)
- [last-build](https://apm-ci.elastic.co/job/jenkins-git-fetch-reference/lastBuild/nodes=master/console)

```
CACHING git@github.com:elastic/apm-server.git -> /var/lib/jenkins/.git-references/apm-server.git
```